Sources/TensorFlow/Initializers.swift

```diff
@@ -459,7 +459,12 @@ fileprivate extension Tensor where Scalar: BinaryFloatingPoint {
         let fanIn = shape[shape.count - 2] * receptiveField
         let fanOut = shape[shape.count - 1] * receptiveField
         let minusOneToOne = 2 * randomUniform - 1
-        return sqrt(Scalar(6) / Scalar(fanIn + fanOut)) * minusOneToOne
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+        let _sqrt = Darwin.sqrt as (Scalar) -> Scalar
+#else
+        let _sqrt = Glibc.sqrt as (Scalar) -> Scalar
+#endif
+        return _sqrt(Scalar(6) / Scalar(fanIn + fanOut)) * minusOneToOne
```

Comment:
This is necessary to avoid an unfortunate type-checking ambiguity:
```
/Users/danielzheng/swift-apis/Sources/TensorFlow/Initializers.swift:462:16: error: referencing static method 'sqrt' on 'Tensor' requires that 'Scalar' conform to 'TensorFlowFloatingPoint'
        return sqrt(Scalar(6) / Scalar(fanIn + fanOut)) * minusOneToOne
               ^
/Users/danielzheng/swift-apis/Sources/TensorFlow/Operators/Math.swift:26:1: note: where 'Scalar' = 'Scalar'
extension Tensor: ElementaryFunctions where Scalar: TensorFlowFloatingPoint {
^
```

Reply:
Instead of using the global libc function, you should use the static ElementaryFunction method instead. That shouldn’t have any ambiguity. cc @stephentyrone

Reply:
Done in https://github.com/tensorflow/swift-apis/pull/200/commits/0c1ff4142130ac7a226e882a7f39ee5cefc13bdb. Changed some constraints to `Scalar: TensorFlowFloatingPoint`.